go.mod

```diff
@@ -147,6 +146,6 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/figment-networks/cosmos-sdk v0.45.1-dm
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
-	github.com/tendermint/tendermint => github.com/figment-networks/tendermint v0.34.14-deepmind
+	github.com/tendermint/tendermint => github.com/figment-networks/tendermint v0.34.10-0.20220406135601-70e7eea733ef
```

Comment:
This tendermint import is wrong, it should always match with the original version https://github.com/figment-networks/gaia/blob/15d2af2d91396f1ddc1dc866a03ac1214aaa00be/go.mod#L18 (but with our -deepmind instrumentation)

Reply:
why? on `deepmind-*` branches we're using forked tendermint branch to run this binary with the extractor. We should create a new tag for `tendermint` and then I'll update this PR

Reply:
Im referring to the version of tendermint that gaia depends on - you're downgrading v0.34.14 to v0.34.10